### PR TITLE
OEUI: Add Order Entry UI OWA to addon index list

### PR DESCRIPTION
**Pull request by larrystone**
**SUMMARY**
Add Order Entry UI OWA to addon index list

Check [here](https://talk.openmrs.org/t/order-entry-ui-sprint-6-announcement/18351) for more information about the Order Entry UI OWA

### CODE REVIEW

File: src/main/resources/add-ons-to-index.json [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
+          "url": "https://andela.com/"
+        },
+        {
+          "name": "OpenMRS",
+          "url": "https://github.com/openmrs"
+        }
+      ],
+      "backend": "org.openmrs.addonindex.backend.Bintray",
+      "bintrayPackageDetails": {
+        "owner": "openmrs",
```
Comment (reubenvarghese1) [inline src/main/resources/add-ons-to-index.json]: According to Bintray, you i.e. larrystone should be the owner. Hence, kindly change this from openmrs to larrystone

File: src/main/resources/add-ons-to-index.json [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
+      "uid": "org.openmrs.owa.orderentryui",
+      "type": "OWA",
+      "name": "Order Entry UI OWA",
+      "maintainers": [
+        {
+          "name": "Andela apprentices",
+          "url": "https://andela.com/"
+        },
+        {
+          "name": "OpenMRS",
```
Comment (reubenvarghese1) [inline src/main/resources/add-ons-to-index.json]: Are you sure we can should OpenMRS as a maintainer?  What do you feel @dkayiwa @djazayeri ?

File: src/main/resources/add-ons-to-index.json (hunk 1 above)
Comment (larrystone) [reply src/main/resources/add-ons-to-index.json]: Thanks for this observation. 
I am currently working on been added to the openmrs bintray account. When it gets approved, I will be creating the package directly on the openmrs owa repo, removing the existing link to mine

Comment (larrystone) [reply src/main/resources/add-ons-to-index.json]: I have been granted the permission and I have created the package on the openmrs organisation's owa repo. @reubenvarghese1

Comment (reubenvarghese1) [reply src/main/resources/add-ons-to-index.json]: Great thanks, I am merging this PR :)

**Pull request merged**